engine/src/test/java/org/hibernate/validator/test/cfg/ConstraintMappingTest.java

```diff
@@ -145,7 +149,7 @@ public void testSingleConstraint() {
 
 		Set<ConstraintViolation<Marathon>> violations = validator.validate( new Marathon() );
 		assertNumberOfViolations( violations, 1 );
-		assertConstraintViolation( violations.iterator().next(), "may not be null" );
+		assertConstraintViolation( violations.iterator().next(), NotNull.class, "may not be null" );
```

Comment:
(Independently of this PR) It'd be great to have this:

    assertSingleConstraintViolation( violations, NotNull.class, "may not be null" );

It avoids the - often not so helpful - assertion on the number of constraints - on the line before.